examples: Error[E0425]: cannot find function `align_of` in this scope

Trying to run the examples on MacOS with the VulkanSDK `1.3.296.0` logs the following error several times:
``` rust
cargo run -p ash-examples --bin texture
   Compiling ash-examples v0.1.0 (/Users/retbleed/Projects/gamedev/ash/ash-examples)
error[E0425]: cannot find function `align_of` in this scope
   --> ash-examples/src/bin/texture.rs:135:13
    |
135 |             align_of::<u32>() as u64,
    |             ^^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
3   + use core::mem::align_of;
    |
3   + use crate::mem::align_of;
    |
3   + use std::mem::align_of;
    |

error[E0425]: cannot find function `align_of` in this scope
   --> ash-examples/src/bin/texture.rs:203:13
    |
203 |             align_of::<Vertex>() as u64,
    |             ^^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
3   + use core::mem::align_of;
    |
3   + use crate::mem::align_of;
    |
3   + use std::mem::align_of;
    |

error[E0425]: cannot find function `align_of` in this scope
   --> ash-examples/src/bin/texture.rs:258:13
    |
258 |             align_of::<Vector3>() as u64,
    |             ^^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
3   + use core::mem::align_of;
    |
3   + use crate::mem::align_of;
    |
3   + use std::mem::align_of;
    |

error[E0425]: cannot find function `align_of` in this scope
   --> ash-examples/src/bin/texture.rs:308:13
    |
308 |             align_of::<u8>() as u64,
    |             ^^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
3   + use core::mem::align_of;
    |
3   + use crate::mem::align_of;
    |
3   + use std::mem::align_of;
    |

For more information about this error, try `rustc --explain E0425`.
error: could not compile `ash-examples` (bin "texture") due to 4 previous errors
```

VulkanSDK Version = `1.3.296.0` 
MacOS Version = `15.0.1`

--- a/ash-examples/src/bin/texture.rs
+++ b/ash-examples/src/bin/texture.rs
@@ -5,7 +5,7 @@ use std::error::Error;
 use std::ffi;
 use std::io::Cursor;
 use std::mem;
-use std::mem::{size_of, size_of_val}; // TODO: Remove when bumping MSRV to 1.80
+use std::mem::{align_of, size_of, size_of_val}; // TODO: Remove when bumping MSRV to 1.80
 use std::os::raw::c_void;
 
 use ash::util::*;

--- a/ash-examples/src/bin/triangle.rs
+++ b/ash-examples/src/bin/triangle.rs
@@ -5,7 +5,7 @@ use std::error::Error;
 use std::ffi;
 use std::io::Cursor;
 use std::mem;
-use std::mem::{size_of, size_of_val}; // TODO: Remove when bumping MSRV to 1.80
+use std::mem::{align_of, size_of, size_of_val}; // TODO: Remove when bumping MSRV to 1.80
 
 use ash::util::*;
 use ash::vk;

